credentials/error.go

```diff
@@ -35,3 +42,73 @@ func IsErrCredentialsNotFound(err error) bool {
 func IsErrCredentialsNotFoundMessage(err string) bool {
 	return err == errCredentialsNotFoundMessage
 }
+
+// errCredentialsMissingServerURL represents an error raised
+// when the credentials object has no server URL or when no
+// server URL is provided to a credentials operation requiring
+// one.
+type errCredentialsMissingServerURL struct{}
+
+// Error returns the standard error message for when
+// the credentials object has no server URL.
+func (errCredentialsMissingServerURL) Error() string {
+	return errCredentialsMissingServerURLMessage
+}
+
+// NewErrCredentialsMissingServerURL creates a new error for
+// when the credentials object has no server URL or when no
+// server URL is provided to a credentials operation requiring
+// one.
+func NewErrCredentialsMissingServerURL() error {
+	return errCredentialsMissingServerURL{}
+}
+
+// IsCredentialsMissingServerURL returns true if the error
+// was caused by not having a credentials server URL when
+// required.
+func IsCredentialsMissingServerURL(err error) bool {
+	_, ok := err.(errCredentialsMissingServerURL)
+	return ok
+}
+
+// IsCredentialsMissingServerURLMessage returns true if the error
+// was caused by not having a credentials server URL when
+// required.
+func IsCredentialsMissingServerURLMessage(err string) bool {
+	return err == errCredentialsMissingServerURLMessage
```

Comment:
You're right, I forgot to add the error checks in the output logs from credentials store client.

Done in : https://github.com/docker/docker-credential-helpers/pull/62/commits/b69480e07432d492cbf5ef409dd80c7124b42796